React/Views/RCTScrollView.m

```diff
@@ -289,8 +289,9 @@ - (void)dockClosestSectionHeader
   [_stickyHeaderIndices enumerateIndexesWithOptions:0 usingBlock:
    ^(NSUInteger idx, __unused BOOL *stop) {
 
+    // If the subviews are out of sync with the sticky header indices don't
+    // do anything.
     if (idx >= subviewCount) {
-      RCTLogError(@"Sticky header index %zd was outside the range {0, %zd}", idx, subviewCount);
```

Comment:
Can we keep the warning for now, but just downgrade it to RCTLogWarn? I'd rather keep it until we're sure of what causes it (there are other cases besides pull to refresh).

Also can you add *stop = YES; before the return as well? Just so we don't generate repeat warnings.